### PR TITLE
fix(VListItem): add anchor title

### PR DESCRIPTION
**Pull request by bogdanciuca**
## Description

`VListItem` anchor is missing the `title` prop. Fixes: https://github.com/vuetifyjs/vuetify/issues/17908

## Markup:

```vue
<template>
  <v-app>
    <v-container>
      <v-card>
        <v-list>
          <v-list-item
            v-for="(item, index) in items"
            :key="index"
            :title="item.title"
            :href="item.href"
          />
        </v-list>
      </v-card>
    </v-container>
  </v-app>
</template>

<script setup>
  import { ref } from 'vue'

  const items = ref([
    {
      title: 'Vuetify',
      href: 'https://vuetifyjs.com/en/',
    },
    {
      title: 'Vue',
      href: 'https://vuejs.org/',
    },
  ])
</script>
```

### CODE REVIEW

File: packages/vuetify/src/components/VList/VListItem.tsx [hunk 1]
```diff
@@ -220,6 +220,7 @@ export const VListItem = genericComponent<VListItemSlots>()({
           ]}
           href={ link.href.value }
           tabindex={ isClickable.value ? (list ? -2 : 0) : undefined }
+          title={ slots.title?.({ title: props.title }) ?? props.title }
```
Comment (johnleider) [inline packages/vuetify/src/components/VList/VListItem.tsx:223]: I've never seen a slot used like this before. It also means you'd have to update the available slots for this component. I think it's safe for this to simply equal props.title.

**Review by johnleider — CHANGES_REQUESTED** [review]

**Review by johnleider — APPROVED** [review]

**Comment by KaelWD** [comment]
I'm reverting this, it adds `title="false"` if the title prop is not set and even if it is set you don't always want a tooltip.

**Comment by johnleider** [comment]
> I'm reverting this, it adds `title="false"` if the title prop is not set and even if it is set you don't always want a tooltip.

Then why not adjust it to be undefined if not set instead of reverting it?

**Comment by KaelWD** [comment]
Because the second part of that sentence.

**Comment by chunlampang** [comment]
Because of `title="false`, when hover the list item, it showing "false" by the default html title attribute.

**Pull request merged**